src/data/blogs.json

```diff
@@ -14,5 +14,17 @@
     {
         "title": "Laravel News Blog",
         "link": "https://laravel-news.com/blog"
+    },
+    {
+        "title": "Laravel Daily",
+        "link": "https://laraveldaily.com"
+    },
+    {
+        "title": "Laravel Tricks",
+        "link": "https://laravel-tricks.com/tricks"
+    },
+    {
+        "title": "Laraship",
```

Comment:
Seems like most of the resource in `laraship` is about their product, not about Laravel. so, it'd be better not to add it in our resource list.

Reply:
Removed laraship from blogs